docs/_documentations/che-installinfo.md

```diff
@@ -7,14 +7,32 @@ duration: 1 minute
 permalink: che-installinfo
 type: document
 ---
+# Using Codewind as a hosted application on the cloud
 
-# Installing Eclipse Che for Codewind
+Codewind can be used in one of three ways - [locally](./vsc-getting-started.html), [remotely](./remote-codewind-overview.html), or **hosted as an IDE extension on the cloud**. By using Codewind hosted as an application on the cloud, you can develop, build and run your code all by accessing Codewind from a browser window from your local desktop.
+
+Follow the instructions to get started with using Codewind as an application hosted on the cloud, which guides you through:
+
+1. Installing Eclipse Che to use with Codewind or update an existing Eclipse Che installation to use Codewind
+2. Configuring Codewind on Eclipse Che to work with your browser and cluster.
+2. Creating and configuring a Codewind workspace:
+- Creating a Codewind workspace in Eclipse Che
+- Adding an image registry in Eclipse Che (optional)
+
+4. Creating your first Codewind project 
+5. Making a code change to try out the inner loop experience 
 
-Install Che to use with Codewind or prepare to use Codewind with an existing Che installation.
+### Other Codewind configurations
+
+**A quick way to try out Codewind before installing Codewind on Eclipse Che is to use Codewind locally** In this fully-local configuration, you create, develop, build and run your containerized applications on your local machine using your local IDE. See [Getting Started with Codewind](./gettingstarted.html) for local IDE options and step-by-step instructions.
+
+**Using Codewind Remotely** Rather than using an IDE hosted in the cloud to make code changes, Codewind can also be configured for making code changes on your **local IDE** before building and running on your cloud. To [configure Codewind to be used remotely](./remote-codewind-overview.html), **you must first follow the steps to [install codewind locally](./gettingstarted.html)**. 
+
```

Comment:
This signposting and orientation will be encased in a callout box to prevent this confusion. Callout box functionality will be delievered to the master branch imminently.